src/wait-for-element-to-be-removed.js

```diff
@@ -1,85 +1,48 @@
-import {
```

Comment:
It's easier to just look at the post-change version of this file: https://github.com/testing-library/dom-testing-library/blob/pr/wait-improvements/src/wait-for-element-to-be-removed.js